src/collections.js

```diff
@@ -39,3 +39,36 @@ export function mapOrNull<T, U>(
   }
   return result;
 }
+
+/**
+ * Find items that are in one but not both of the given arrays.
+ *
+ * The two returned arrays consist of items in only the first array, and
+ * items in only the second array, respectively.
+ *
+ * Both input arrays must be sorted.  The output arrays will also be sorted.
```

Comment:
> I think the requirement is specifically that they're sorted ascending, is that right?

Yep. That's what "sorted" conventionally means when not otherwise specified.

In our codebase, searching for "sorted" finds a number of other places we say things are "sorted" (or "sorted by" some key, etc.), and they have the same meaning.



> But do you think future callers might not be as well-behaved, or prone to change so they violate the invariant? What do you think about a dev-only error?

Good question. I think that's possible but not an especially high risk, and this isn't a case where we need such a debug check.